include/zephyr/drivers/bluetooth.h

```diff
@@ -87,6 +87,7 @@ enum bt_hci_bus {
 #define BT_DT_HCI_NAME_GET(node_id) DT_PROP_OR(node_id, bt_hci_name, "HCI")
 #define BT_DT_HCI_NAME_INST_GET(inst) BT_DT_HCI_NAME_GET(DT_DRV_INST(inst))
 
+#define BT_HCI_BUS_VIRTUAL (0)
```

Comment:
Hmm, isn't there a risk of some static analyzer warning about this since the enum has the same name? Another option would have been:
```c
/* Fallback default when there's no property, same as "virtual" */
#define BT_HCI_BUS_DEFAULT 0
```